### PR TITLE
Creating_settings_route

### PR DESCRIPTION
**Pull request by MartaSzuran**

### CODE REVIEW

File: app/home/user-settings/controller.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+export default class HomeUserSettingsController extends Controller {
+  @service store;
+  @service router;
+
+  @tracked username;
+  @tracked email;
+  @tracked info;
+  @tracked password;
+  @tracked photoURL;
+  @tracked photo = this.model.photoURL;
```
Comment (wozny1989) [inline app/home/user-settings/controller.js]: Declare empty `@tracked` and set in `constructor`

File: app/home/user-settings/controller.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+  @action
+  onPasswordChange({ target: { value } }) {
+    this.model.password = value;
+  }
+
+  @action
+  onPhotoURLChange({ target: { value } }) {
+    this.model.photoURL = value;
+  }
```
Comment (wozny1989) [inline app/home/user-settings/controller.js]: These actions are almost identical. Consider migrate to one, common action `onPropetyChange` or similar which take model key and override it value

**Review by wozny1989 — CHANGES_REQUESTED** [review]
🚀

File: app/home/user-settings/controller.js (hunk 1 above)
Comment (MartaSzuran) [reply app/home/user-settings/controller.js]: It has turned out that I don't need that declaration.

File: app/home/user-settings/controller.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+export default class HomeUserSettingsController extends Controller {
+  @service store;
+  @service router;
+  @tracked photo;
+
+  @action
+  onPropertyChange(propName, { target: { value } }) {
+    this.model[propName] = value;
+  }
```
Comment (wozny1989) [inline app/home/user-settings/controller.js:14]: 🚀

**Review by wozny1989 — APPROVED** [review]

**Pull request merged**